UnitTest/TestComponentCSharp_Tests.net5.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.IO;
+using TestComponentCSharp;
+using Windows.Storage;
+using Xunit;
+
+namespace UnitTest
+{
+    public class TestCSharpNet5
```

Comment:
Any particular reason for putting this test in a separate file and the net5 suffx?  Anything net5 specific here?

Reply:
OpenStreamForReadAsync is a net5 function, it isn't available in net2